include/aws/common/common.h

```diff
@@ -223,6 +229,40 @@ static inline void aws_secure_zero(void *pBuf, size_t bufsize) {
 
 #define AWS_ZERO_STRUCT(object) memset(&object, 0, sizeof(object));
 #define AWS_ZERO_ARRAY(array) memset((void *)array, 0, sizeof(array));
+#define AWS_PTR_ADD(ptr, bytes) ((void *)(((char *)ptr) + (bytes)))
```

Comment:
char is technically not guaranteed to only be 1 byte. I'd recommend uint8_t

Reply:
Good catch, thanks!